tests/e2e/cucumber/features/integrations/link.feature

```diff
@@ -4,7 +4,7 @@ Feature: link
     Given "Admin" creates following users
       | id    |
       | Alice |
-    When "Alice" logs in
+    When "Alic" logs in
```

Comment:
:question:

Reply:
I am checking the workflow :sweat_smile: